src/playbacks/hls/hls.js

```diff
@@ -124,6 +123,12 @@ export default class HLS extends HTML5VideoPlayback {
     }
   }
 
+  render() {
+    var a = super.render()
```

Comment:
what's `a` ? is it super.element (video tag element) ? If yes, I think we should name this to `videoElement` or something that show intention